src/Uno.UI/UI/Xaml/Media/Animation/Timeline.cs

```diff
@@ -102,6 +102,8 @@ void ITimeline.UnregisterListener(ITimelineListener listener)
 
 		protected void OnCompleted()
 		{
+			State = TimelineState.Stopped;
```

Comment:
I see code that intentionally set `State` to other values before calling `OnCompleted`.

https://github.com/unoplatform/uno/blob/46c179643b3e94ff95b867db3996f481e6c15789/src/Uno.UI/UI/Xaml/Media/Animation/DoubleAnimationUsingKeyFrames.cs#L326-L337

So I'm not sure if this change is correct. Thoughts @jeromelaban ?

Reply:
@dr1rrb will have more insights on this change

Reply:
It seems that animations using KeyFrames set the State to TimeLineState.Stopped when reach OnEnd method, but others animations like ColorAnimation or DoubleAnimation that doesn't use KeyFrames, never set the State to Stopped. I think that Timeline.OnComplete is a good place to change the State.

Reply:
Well I'm not sure of this specific `State` property (seems to be an internal property that is designed to be used for the [`StoryBoard.GetCurrentState`](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.media.animation.storyboard.getcurrentstate)).

But @Youssef1313 comment's is relevant and looking at the MS docs and the code of the KeyFrame animation, it seems that this property should not be set to `Stopped` each time we invoke the `OnCompleted`. We should instead replicate / reuse the `OnEnd` from the KeyFrame to all other animations kind.

Reply:
I've explained this in the linked issue, but I will copy here again:
> It is normal that the `Storyboard::GetCurrentState()` is not `ClockState.Stopped` when `Timeline::Completed` occurs. Because by default the `Timeline::FillBehavior` is `HoldEnd`, which allow the animated value to "stay" after the animation is completed and the timeline will be in a `ClockState.Filling` state on completion.
This is exactly the behavior we are getting from windows, so there is nothing to change here.

Furthermore, this only fix the symptom of the problem, not the cause. The OP from the linked issue want to modify the animated value after the animation is completed, but is unable to do so because of (#631). Hence why he is trying to `Stop()` the storyboard after completion.
Now merely change the state to `Stopped`, isn't going to fix the underlying problem.